bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4578,134 +4578,148 @@
         </sql>
     </changeSet>
 
-    <changeSet id="bahmni-emrapi-202208101339" author="BAHMNI" >
+
+    <changeSet id="bahmni-emrapi-202208101439" author="BAHMNI" >
         <preConditions onFail="MARK_RAN">
             <sqlCheck expectedResult="1">
                 select count(*) from concept c
                 INNER join concept_name cn
-                on cn.concept_id=c.concept_id and cn.name='Visit Diagnoses' and cn.concept_name_type='FULLY_SPECIFIED' and cn.voided=0 and c.version is null;
+                on cn.concept_id=c.concept_id and cn.name='Visit Diagnoses' and cn.concept_name_type='FULLY_SPECIFIED' and cn.voided=0 and cn.locale = 'en' and c.version is null;
             </sqlCheck>
             <sqlCheck expectedResult="1">
+                <!-- The below query returns the concept which has only one concept reference mapping !-->
                 select count(*) from concept_reference_map rmap, concept_name cn where cn.concept_id = rmap.concept_id and cn.name='Visit Diagnoses'
```

Comment:
instead of count(*), use specific - either rmap.<id> 
Also pref INNER JOIN .. its more explicit.

Reply:
Done